.agents/skills/trigger-agents/SKILL.md

```
@@ -0,0 +1,287 @@
+---
+name: trigger-agents
+description: AI agent patterns with Trigger.dev - orchestration, parallelization, routing, evaluator-optimizer, and human-in-the-loop. Use when building LLM-powered tasks that need parallel workers, approval gates, tool calling, or multi-step agent workflows.
+---
+
+# AI Agent Patterns with Trigger.dev
+
+Build production-ready AI agents using Trigger.dev's durable execution.
+
+## Pattern Selection
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)